num_list.py

```diff
@@ -7,9 +7,11 @@ def print_each_number():
     # 10
     # ...etc
     # TODO - Write your code below this line.  (Keep the indentation)
-    
 
-    
+for i in nums:
```

Comment:
Check your indentation on this part of your code, could be what is causing the failed tests.